### PR TITLE
Add function to retrieve files from Appwrite storage and upload them …

### PR DESCRIPTION
**Pull request by geryjuhasz**
…to Dropbox

### CODE REVIEW

**Comment by geryjuhasz** [comment]
out

logs

db

File: php/file-backup/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# 📧 Get Files from Appwrite storage and upload them to Dropbox
+A sample PHP Cloud Function for uploading files to Dropbox
+
+## 📝 Environment Variables
+Make sure to set the following environment variables.
+
+* **APPWRITE_ENDPOINT** - AppWrite Domain 
+* **APPWRITE_FUNCTION_PROJECT_ID** - AppWrite Project ID
```
Comment (christyjacob4) [inline php/file-backup/README.md:8]:
```suggestion
```
you can remove this var. it's passed by default to the cloud function

File: php/file-backup/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# 📧 Get Files from Appwrite storage and upload them to Dropbox
+A sample PHP Cloud Function for uploading files to Dropbox
+
+## 📝 Environment Variables
+Make sure to set the following environment variables.
+
+* **APPWRITE_ENDPOINT** - AppWrite Domain
```
Comment (christyjacob4) [inline php/file-backup/README.md:7]:
```suggestion
* **APPWRITE_ENDPOINT** - Appwrite Domain 
```

File: php/file-backup/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# 📧 Get Files from Appwrite storage and upload them to Dropbox
+A sample PHP Cloud Function for uploading files to Dropbox
+
+## 📝 Environment Variables
+Make sure to set the following environment variables.
+
+* **APPWRITE_ENDPOINT** - AppWrite Domain 
+* **APPWRITE_FUNCTION_PROJECT_ID** - AppWrite Project ID
+* **APPWRITE_API_KEY** - AppWrite API key
```
Comment (christyjacob4) [inline php/file-backup/README.md:9]:
```suggestion
* **APPWRITE_API_KEY** - Appwrite API key
```

File: php/file-backup/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+* Ensure that your folder structure looks like this 
+```
+.
+├── index.php
+├── vendor/
+├── composer.json
+└── composer.lock
+└── README.md
+```
+
```
Comment (christyjacob4) [inline php/file-backup/README.md:31]: You're missing the steps required to package this as a tarfile. Please take a look at the other examples to find how.

File: php/file-backup/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+## 🎯 Trigger
+
+```bash 
+php index.php fileId1 fileId2 ...
```
Comment (christyjacob4) [inline php/file-backup/README.md:35]: The trigger section does not look right. Please refer to my comment from earlier https://github.com/appwrite/demos-for-functions/pull/244#discussion_r757902352

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please address the comments

**Comment by christyjacob4** [comment]
@geryjuhasz  closing this PR for now . Feel free to reopen it / create a new one when you're ready to start working on it once again

**Pull request closed without merging**